rest/client/client.go

```diff
@@ -181,55 +179,62 @@ func (c *communicatorImpl) GetLoggerProducer(ctx context.Context, taskData TaskD
 	}
 }
 
-func (c *communicatorImpl) makeSender(ctx context.Context, taskData TaskData, opts LogOpts, prefix string) send.Sender {
+func (c *communicatorImpl) makeSender(ctx context.Context, taskData TaskData, opts []LogOpts, prefix string) send.Sender {
 	levelInfo := send.LevelInfo{Default: level.Info, Threshold: level.Debug}
-	var sender send.Sender
-	var err error
-	bufferDuration := defaultLogBufferTime
-	if opts.BufferDuration > 0 {
-		bufferDuration = opts.BufferDuration
-	}
-	bufferSize := defaultLogBufferSize
-	if opts.BufferSize > 0 {
-		bufferSize = opts.BufferSize
-	}
-	switch opts.Sender {
-	case model.FileLogSender:
-		sender, err = send.NewPlainFileLogger(prefix, opts.Filepath, levelInfo)
-		if err != nil {
-			grip.Critical(errors.Wrap(err, "error creating file logger"))
-			return nil
-		}
-		sender = send.NewBufferedSender(sender, bufferDuration, bufferSize)
-	case model.SplunkLogSender:
-		info := send.SplunkConnectionInfo{
-			ServerURL: opts.SplunkServerURL,
-			Token:     opts.SplunkToken,
+	senders := []send.Sender{grip.GetSender()}
```

Comment:
Why set a sender in this array before adding to it?

Reply:
This is a slight refactor of the previous behavior. We always want to log to the local (agent log file) sender in addition to the sender we actually care about

Reply:
because we mirror everything to the agent's local logger/log file.

arguably we should *not* do this and then maybe splunk all agent logs.